javascript_nodejs/1.console-echobot/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "console-echobot",
+  "version": "1.0.0",
+  "description": "Bot Builder v4 console echobot sample",
+  "main": "index.js",
+  "scripts": {
+    "start": "node ./index.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

Comment:
For consistency, we should add "watch": "nodemon ./index.js"

Reply:
OK!